_docs/frida-trace.md

```diff
@@ -7,12 +7,189 @@ permalink: /docs/frida-trace/
+-U, --usb             connect to USB device
+-R, --remote          connect to remote frida-server
+-H HOST, --host=HOST  connect to remote frida-server on HOST
+-f FILE, --file=FILE  spawn FILE
+-F, --attach-frontmost
+                      attach to frontmost application
+-n NAME, --attach-name=NAME
+                      attach to NAME
+-p PID, --attach-pid=PID
+                      attach to PID
+--stdio=inherit|pipe  stdio behavior when spawning (defaults to "inherit")
+--runtime=duk|v8      script runtime to use (defaults to "duk")
+--debug               enable the Node.js compatible script debugger
+-I MODULE, --include-module=MODULE
+                      include MODULE
+-X MODULE, --exclude-module=MODULE
+                      exclude MODULE
+-i FUNCTION, --include=FUNCTION
+                      include FUNCTION
+-x FUNCTION, --exclude=FUNCTION
+                      exclude FUNCTION
+-a MODULE!OFFSET, --add=MODULE!OFFSET
+                      add MODULE!OFFSET
+-T, --include-imports
+                      include program's imports
+-t MODULE, --include-module-imports=MODULE
+                      include MODULE imports
+-m OBJC_METHOD, --include-objc-method=OBJC_METHOD
+                      include OBJC_METHOD
+-M OBJC_METHOD, --exclude-objc-method=OBJC_METHOD
+                      exclude OBJC_METHOD
+-s DEBUG_SYMBOL, --include-debug-symbol=DEBUG_SYMBOL
+                      include DEBUG_SYMBOL
+-q, --quiet           do not format output messages
+-d, --decorate        Add module name to generated onEnter log statement
+-o OUTPUT, --output=OUTPUT
+                      dump messages to file
+{% endhighlight %}
+
+## -U, --usb: connect to USB device
+
+This option tells `frida-trace` to perform tracing on a remote device 
+connected via the host machine's USB connection.
+
+Example: You want to trace an application running on an Android device
+from your host Windows machine.  If you specify `-U / --usb`, 
+frida-trace will perform the necessary work to transfer all data to 
+and from the remote device and trace accordingly.
+
+<div class="note">
+  <h5>Copy frida-server binary to remote device</h5>
+  <p>When tracing a remote device, remember to copy the 
+  <a href="https://github.com/frida/frida/releases">platform-appropriate frida-server binary</a>
+  to the remote device.  Once copied, be sure to run the frida-server binary before
+  beginning the tracing session.</p>
+  <p>For example, to trace a remote Android application, you would copy the 
+  'frida-server-12.8.0-android-arm' binary to the Android's /data/local/tmp 
+  folder.  Using adb shell, you would run the server in the background 
+  (e.g. frida-server-12.8.0-android-arm &).</p>
+</div>
+
+## -I, -X: include/exclude module
+
+These options allow you to include or exclude **all** functions in a particular 
+module (e.g., *.so, *.dll) in one, single option.  The option expects a regular
+expression for matching one or more modules.  Any module that matches the
```

Comment:
We only support globs:
https://github.com/frida/frida-gum/blob/5858a8fecb291c3e60c300191ef156c8c4b36f77/gum/gummoduleapiresolver.c#L145
As it's built on [GLib.PatternSpec](https://developer.gnome.org/glib/stable/glib-Glob-style-pattern-matching.html).

Reply:
Spot on comment, done.